Core/include/Acts/Propagator/EigenStepper.ipp

```diff
@@ -286,6 +291,18 @@ Acts::Result<double> Acts::EigenStepper<E>::step(
     }
   }
 
+  state.stepping.pathAccumulated += h;
```

Comment:
This confused me a bit. This is not for statistics strictly speaking. Why do we do this before the Cov transport now? In general with the statistics, do we want to include the numbers even if the cov transport branch can terminate? Just wondering what the most unsurprising location for the counting is.

Reply:
True I can move that back